bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/events/RuleRemovedEvent.java

```diff
@@ -44,7 +43,7 @@ public String getType() {
 
     @Override
     public String toString() {
-        return "Rule '" + getRule().uid + "' has been removed.";
+        return "RuleImpl '" + getRule().uid + "' has been removed.";
```

Comment:
Hm, why "RuleImpl" instead of "Rule"?

Reply:
I had the same issue in my previous comment, I will update the pull request.